pages/index.js

```diff
@@ -1,17 +1,46 @@
-import Navbar from "@/components/pages/home/Navbar";
 import Head from "next/head";
+import Navbar from "@/components/pages/home/Navbar";
 import HomeHeader from "@/components/pages/home/HomeHeader";
+import MemberCard from "@/components/pages/home/MemberCard";
```

Comment:
Diubah path importnya

```suggestion
import OurTeam from "@/components/pages/home/OurTean";
```